src/DeploymentEnvironments.ts

```diff
@@ -0,0 +1,50 @@
+import { Environment } from 'aws-cdk-lib';
+
+export interface DeploymentEnvironment {
+  accountName: string;
+  env: Environment;
+  assumedRolesToAlarmOn?: string|string[];
+}
+
+/**
+ * List all environments for which which a monitoring
+ * pipeline should be deployed in prod
+ */
+export const deploymentEnvironments: DeploymentEnvironment[] = [
+  {
+    accountName: 'auth-accp',
+    env: {
+      account: '315037222840',
+      region: 'eu-west-1',
+    },
+  },
+  {
+    accountName: 'auth-prod',
+    env: {
+      account: '196212984627',
+      region: 'eu-west-1',
+    },
+  },
+  {
+    accountName: 'dns',
+    env: {
+      account: '108197740505',
+      region: 'eu-west-1',
+    },
+    assumedRolesToAlarmOn: 'nijmegen-operator',
+  },
+  {
+    accountName: 'production',
```

Comment:
production van wat? Welk account?

Reply:
de account zelf heet 'production'. Deze naam wordt ook gebruikt om bijv de cloudtrail-loggroup te vinden, dus moet gelijk zijn aan de naam die we (in terraform/alias) gebruikt hebben voor de account. Dat dit een matige naam is in onze huidige opzet ben ik het mee eens :).

Reply:
Ah die oké